<!DOCTYPE html>
<html>

<style>
seq {font-family:Monaco, "Courier New";
	font-size:0.9em;
	text-indent: 20px;
}

align.small {
    font-family:Monaco, "Courier New";
    font-size:0.7em;
    line-height:65%;
}

align.big {
	font-family:Monaco, "Courier New";
	font-size:1em;
	line-height:100%;
}

s5 {color:#6baed6;}
s7 {color:#fc9272;}
p5 {color:#08519c;}
p7 {color:#a50f15;}
me {color:#969696;}
t7 {color:blue;}
umi {color:#807dba;}
cbc {color:#f768a1;}
tso {color:#2ca25f;}
h3 {font-family:verdana;}

</style>

<head>
<title>G&T-seq</title>
</head>
<body>

<h1><a href="https://www.nature.com/articles/nmeth.3370" target="_blank">G&T-seq</a></h1>

<p><span style="font-family:verdana; font-size:1.1em;">The genome and transcriptome sequencing (G&T-seq) uses oligo-dT beads to physically separate mRNA and gDNA from single celss. For mRNA-seq, it uses the <a href="https://teichlab.github.io/scg_lib_structs/methods_html/SMART-seq_family.html" target="_blank">SMART-seq</a> method to construct sequencing library; for gDNA, you can choose <a href="https://teichlab.github.io/scg_lib_structs/methods_html/MALBAC.html" target="_blank">MALBAC</a>, <a href="https://teichlab.github.io/scg_lib_structs/methods_html/LIANTI.html" target="_blank">LIANTI</a> or other methods to construct gDNA sequencing library. Click the links to those methods to see how exactly libraries are made.</span></p>

<img src="../data/G_and_T_seq.png" style="height:700px">

</body>
</html>
