<!DOCTYPE html>
<html>

<style>
seq {font-family:Monaco, "Courier New";
	font-size:0.9em;
	text-indent: 20px;
}

align.small {
    font-family:Monaco, "Courier New";
    font-size:0.7em;
    line-height:65%;
}

align.big {
	font-family:Monaco, "Courier New";
	font-size:1em;
	line-height:100%;
}

s5 {color:#6baed6;}
s7 {color:#fc9272;}
p5 {color:#08519c;}
p7 {color:#a50f15;}
me {color:#969696;}
t7 {color:blue;}
umi {color:#807dba;}
cbc {color:#f768a1;}
tso {color:#2ca25f;}
h3 {font-family:verdana;}

</style>

<head>
<title>scCAT-seq</title>
</head>
<body>

<h1><a href="https://www.nature.com/articles/s41467-018-08205-7" target="_blank">scCAT-seq</a></h1>

<p><span style="font-family:verdana; font-size:1.1em;">The single cell chromatin accessibility and transcriptome sequencing (scCAT-seq) physically separates mRNA and chromatin from single cells. The method uses gentle lysis to release mRNA into the supernatant, then transfer supernatant to new tubes and leave nuclei at the original tube. After that, mRNA and nulcei are processed separately. For mRNA-seq, it uses the <a href="https://teichlab.github.io/scg_lib_structs/methods_html/SMART-seq_family.html" target="_blank">SMART-seq</a> method to construct sequencing library; for chromatin accessibility, it uses standard ATAC-seq workflow to construct sequencing library. Click the links to those methods in the main page to see how exactly libraries are made.</span></p>

<img src="../data/scCAT-seq.png">

</body>
</html>
