<!DOCTYPE html>
<html>

<style>
seq {font-family:Monaco, "Courier New";
  font-size:0.9em;
  text-indent: 2.7em;
}

align.small {
    font-family:Monaco, "Courier New";
    font-size:0.9em;
    line-height:65%;
}

align.long {
  font-family:Monaco, "Courier New";
  font-size:0.8em;
  line-height:70%;
}

s5 {color:#6baed6;}
s7 {color:#fc9272;}
p5 {color:#08519c;}
p7 {color:#a50f15;}
me {color:#969696;}
tso {color:#2ca25f;}
cbc {color:#f768a1;}
umi {color:#807dba;}
t7 {color:blue;}
pe1 {color:#bcbddc;}
pe2 {color:#9ebcda;}
w1 {color:#f03b20;}
h3 {font-family:verdana;}

</style>

<head>
<title>scMT-seq</title>
</head>
<body>

<h1><a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-0950-z" target="_blank">scMT-seq</a></h1>

<p><span style="font-family:verdana; font-size:1.1em;">The scMT-seq method was published by Hu <i>et al.</i> in Genome Biology 17, 88. This should not to be confused with <a href="https://www.nature.com/articles/nmeth.3728" target="_blank">scM&T-seq</a>. The scMT-seq physically separates nuclei and the cytopalsm. The mRNA is in cytoplasm and <a href="https://www.nature.com/nmeth/journal/v10/n11/full/nmeth.2639.html" target="_blank">SMART-seq2</a> method was used for scRNA-seq, <a href="https://teichlab.github.io/scg_lib_structs/methods_html/SMART-seq_family.html" target="_blank">click here</a> to see the step-by-step protocol; the genomic DNA is in the nuclei, <a href="https://www.nature.com/articles/nprot.2015.039" target="_blank">scRRBS</a> method was used to profile methylation, and <a href="https://teichlab.github.io/scg_lib_structs/methods_html/scRRBS.html" target="_blank">click here</a> to see the step-by-step protocol.</span></p>

<img src="../data/scMT.png" style="width:500px;">

</body>
</html>
